neuror/cut_plane/legacy_detection.py

```diff
@@ -0,0 +1,54 @@
+'''Module for the legacy cut plane detection.
+
+As implemented in:
+https://bbpcode.epfl.ch/source/xref/platform/BlueRepairSDK/BlueRepairSDK/src/repair.cpp#263
+'''
+import logging
+
+from neurom import iter_sections
+from neurom.core import Tree
+from neurom.core.dataformat import COLS
+
+L = logging.getLogger(__name__)
+
+
+def cut_detect(neuron, axis):
```

Comment:
I have chosen to stick to the old implementation as much as possible. Don't be surprised if the code does not look very idiomatic.